tests/snippets/test_series.py

```diff
@@ -418,3 +418,7 @@ def test_types_eq() -> None:
     res1: pd.Series = s1 == 1
     s2 = pd.Series([1, 2, 4])
     res2: pd.Series = s1 == s2
+
+
+def test_types_rename() -> None:
+    s = pd.Series([1, 2, 3]).rename("A")
```

Comment:
Could you please add few more test cases for other possible values on rename? That would be great!

Reply:
Added tests for each type of input and one for the `inplace=True` overload.